.github/workflows/release.yml

```diff
@@ -45,10 +55,16 @@ jobs:
         git config --global user.email "github-actions[bot]@users.noreply.github.com"
         git config --global user.name "github-actions[bot]"
 
-    - name: Bump package version
-      run: |
-        echo "NEW_VERSION=$(npm --no-git-tag-version version ${{ github.event.inputs.release-type }})" >> $GITHUB_ENV
-        echo "RELEASE_TAG=latest" >> $GITHUB_ENV
+    - name: Bump package version (patch)
```

Comment:
I'd leave the "(patch)" out to not pretend we're still doing semver.

Reply:
Removed.